tutorials/vue/v-else-if.md

```diff
@@ -0,0 +1,57 @@
+The `v-else-if` is another directive provided by Vue.
```

Comment:
No passive voice.

"Vue provides a `v-else-if` directive that you can use with `v-if` analogous to using `else if {}` with `if {}`"